src/lib/support/logging/CHIPLogging.cpp

```diff
@@ -91,6 +91,7 @@ static const char ModuleNames[] = "-\0\0" // None
                                   "SPL"   // SetupPayload
                                   "SVR"   // AppServer
                                   "DIS"   // Discovery
+                                  "DSV"   // DiscoveryVerbose
```

Comment:
Why not ChipLogDetail ?

This seems to set an unusual precedent: having some verbosity flags as a separate module.

Reply:
The rationale for this comes from one of Tennessee's previous comments. Copying here: "_Furthermore, a new logging module like DiscoveryVerbose could be added, so that it can be disabled if needed. This will avoid embedded targets from all using this code by default, and building it by default._"

Reply:
After discussing with andy31415 on Slack, removed the DiscoveryVerbose logging module.